templates/rhsm-katello-reconfigure.erb

```diff
@@ -83,6 +83,8 @@ else
       --server.port="$PORT" \
       --rhsm.repo_ca_cert="%(ca_cert_dir)s$KATELLO_SERVER_CA_CERT" \
       --rhsm.baseurl="$BASEURL"
+    subscription-manager config --rhsm.package_profile_on_trans=1 &> /dev/null
```

Comment:
is the `&` needed? i think it would be OK to just let it run and complete without backgrounding it

Reply:
So older versions of subscription manager will not support these options. Moreover subscription manager errors out if it finds a config entry that doesn't exist. For example
```
# subscription-manager config --rhsm.FOO=1 
Usage: subscription-manager config [OPTIONS]

subscription-manager: error: no such option: --rhsm.FOO
```
I do not want the entire script to fail if 'rhsm.package_profile_on_trans' is not supported.
The best way I saw to accomplish this is to do the  ```&> /dev/null`` so that it continues along on failure.

Reply:
@parthaa that sounds good to me. Can you add a comment above saying its OK if the command fails? that way people wont get confused if they are debugging it later on.

Reply:
Updated